README.md

```diff
@@ -38,6 +38,8 @@ A curated list of AWESOME blogs, videos, tutorials, code, tools, scripts... anyt
 - [Design An Enterprise Governance Strategy](https://docs.microsoft.com/en-us/learn/modules/enterprise-governance/)
 - [Design Governance](https://docs.microsoft.com/en-us/learn/modules/design-governance/)
 - [Intro to Azure Policy](https://docs.microsoft.com/en-us/learn/modules/intro-to-azure-policy/)
+- [Manage security operations in Azure](https://docs.microsoft.com/en-us/learn/paths/manage-security-operations/)
+- [Explore security policies and threat protection capabilities in Microsoft Defender for Office 365](https://docs.microsoft.com/en-us/learn/modules/defender-office-365-explore/)
```

Comment:
Hey @ElYusubov this learn module is for M365, and not applicable to Azure Policy. Should it still be in this list?

https://docs.microsoft.com/en-us/learn/modules/defender-office-365-explore/

Reply:
Yes, you ae spot on.
I included it because of section on "Overview of preset security policies and message overrides".
However, I can exclude that if it does not fit well.

Reply:
```suggestion
```

Because it's related to a different solution let's leave that one out of the list.